docs/Documentation.md

```diff
@@ -470,6 +471,80 @@ let WebApp =
     >=> text "pong"
 ```
 
+### fsharp_max_newline_infix_operator_expression_number_of_items
+
+Control the maximum number of certain infix operators for which infix expression
+can be on one line. Default = 1. Requires
+`fsharp_newline_infix_operator_expression_multiline_formatter` to be
+`number_of_items` to take effect. The next entry below contains more details.
+
+`defaultConfig`
+
+```fsharp
+let fourthPower =
+    [ 1; 2; 3 ]
+    |> List.map (fun x -> x * x)
+    >>= (fun x -> [ x * x ])
+
+let eigthPower =
+    [ 1; 2; 3 ]
+    |> List.map (fun x -> x * x)
+    >>= (fun x -> [ x * x ])
+    |> List.choose (fun x -> Some (x * x))
+```
+
+`{ defaultConfig with MaxNewlineInfixOperatorExpressionNumberOfItems = 2; NewlineInfixOperatorExpressionMultilineFormatter = 2 }`
```

Comment:
`NewlineInfixOperatorExpressionMultilineFormatter = 2` should be `NewlineInfixOperatorExpressionMultilineFormatter  = NumberOfItems` here right? Or am I missing something?